### ISSUE TITLE
NullRefException in CatchEncounter

### ISSUE THREAD

**Issue by J-Saas** [FeroxRev/Pokemon-Go-Rocket-API#286]
Log:

```
Exception: System.NullReferenceException: Object reference not set to an instance of an object.
   at PokemonGo.RocketAPI.Logic.Logic.<CatchEncounter>d__9.MoveNext() in Pokemon-Go-Rocket-API\PokemonGo.RocketAPI.Logic\Logic.cs:line 145
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.GetResult()
   at PokemonGo.RocketAPI.Logic.Logic.<ExecuteCatchAllNearbyPokemons>d__8.MoveNext() in Pokemon-Go-Rocket-API\PokemonGo.RocketAPI.Logic\Logic.cs:line 134
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.GetResult()
   at PokemonGo.RocketAPI.Logic.Logic.<ExecuteFarmingPokestopsAndPokemons>d__7.MoveNext() in Pokemon-Go-Rocket-API\PokemonGo.RocketAPI.Logic\Logic.cs:line 112
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.GetResult()
   at PokemonGo.RocketAPI.Logic.Logic.<PostLoginExecute>d__5.MoveNext() in Pokemon-Go-Rocket-API\PokemonGo.RocketAPI.Logic\Logic.cs:line 65
```

**Comment by FeroxRev** [comment]
Can you recheck with latest commits? Create new issue if persists

**Comment by J-Saas** [comment]
Seems fixed in commit 0207d30, have yet to encounter the issue.
